controllers/articlesController.js

```diff
@@ -49,16 +49,31 @@ export default class ArticlesController {
    */
   static async getAll(req, res) {
     try {
-      /**
-       * @todo Page count for pagination
-       */
-      const allArticles = await models.Article.findAll({
-        limit: 10,
+      // TODO: Implement search algorithm here
+      const current = req.body.offset === 0 ? 1 : Number(req.query.page);
+      const { offset } = req.body;
+      const allArticles = await models.Article.findAndCountAll({
+        limit: PAGE_LIMIT,
```

Comment:
#### Nitpick
- Is the page limit static? If yes, then use the static value if one was not provided in the request

Reply:
Yes @aanchirinah, the page limit is static with a value of 20 as specified in the README file. I assigned it to the PAGE_LIMIT constant because it was used in over 10 times in 3 files. I wanted to make sure that changes to the page limit would be quick to implement.

Reply:
Yes but I still want it to be overridden if a limit is supplied in the request query @valenstical

Reply:
Oh, that makes sense. Thank you @aanchirinah, I will modify the code to implement the feedback.